contracts/src/v0.8/KeeperRegistry.sol

```diff
@@ -689,7 +694,7 @@ contract KeeperRegistry is
   /**
    * @inheritdoc MigratableKeeperRegistryInterface
    */
-  function receiveUpkeeps(bytes calldata encodedUpkeeps) external override {
+  function receiveUpkeeps(bytes calldata encodedUpkeeps) external override whenNotPaused {
```

Comment:
We should have same logic as registerUpkeep, so maybe it makes sense to add to registerUpkeep too?